www/docs/intro.md

```diff
@@ -517,6 +517,8 @@ module.exports = {
 
 ### Paths
 
+Provide absolute paths when using the plugin. If a relative path is provided, it will be resolved relative to the root of the project.
```

Comment:
Maybe `Prefer providing absolute` instead of `Provide absolute` ?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/317/commits/f8edb2c0f52589ea5e309a3e67e6b6a289fb6a5c